src/app/ml-training-client.ts

```diff
@@ -0,0 +1,96 @@
+import { type RpcOptions } from '@improbable-eng/grpc-web/dist/typings/client.d';
+import { MLTrainingServiceClient } from '../gen/app/mltraining/v1/ml_training_pb_service';
+import pb from '../gen/app/mltraining/v1/ml_training_pb';
+import { promisify } from '../utils';
+
+type ValueOf<T> = T[keyof T];
+export const { ModelType } = pb;
+export type ModelType = ValueOf<typeof pb.ModelType>;
+export const { TrainingStatus } = pb;
+export type TrainingStatus = ValueOf<typeof pb.TrainingStatus>;
+
+export class MlTrainingClient {
+  private service: MLTrainingServiceClient;
+
+  constructor(serviceHost: string, grpcOptions: RpcOptions) {
+    this.service = new MLTrainingServiceClient(serviceHost, grpcOptions);
+  }
+
+  async submitTrainingJob(
+    datasetId: string,
+    orgId: string,
```

Comment:
Can we make `orgId` the first parameter? I feel like organization should always be first, since that's the main way we separate information